src/util/dpe-to-slate/index.js

```diff
@@ -58,32 +58,30 @@ const convertDpeToSlate = transcript => {
   }
 
   const { words, paragraphs } = transcript;
-  return paragraphs.map(paragraph => {
-    const wordsFiltered = words.filter(word => {
-      if (word.start >= paragraph.start && word.end <= paragraph.end) {
-        return word;
-      }
-    });
 
-    const lastWordIndex = words.length - 1;
-    const lastWordStartTime = words[lastWordIndex].start;
-    const totalTimingsInt = generatePreviousTimings(lastWordStartTime);
-    const text = wordsFiltered
-      .map(w => {
-        return w.text;
-      })
-      .join(' ');
+  const generateText = (paragraph, words) =>
+    words
+      .filter(
+        (word) => word.start >= paragraph.start && word.end <= paragraph.end
+      )
+      .map((w) => w.text)
+      .join(" ");
 
-    return {
-      speaker: paragraph.speaker,
-      start: paragraph.start,
-      previousTimings: generatePreviousTimingsUpToCurrent(totalTimingsInt, paragraph.start),
-      // pre-computing the display of the formatting here so that it doesn't need to convert it in leaf render
-      startTimecode: shortTimecode(paragraph.start),
-      type: 'timedText',
-      children: [{ text }],
-    };
-  });
+  const generateTotalTimings = (words) =>
+    generatePreviousTimings(words[words.length - 1].start);
```

Comment:
getting tripped up by the syntax, is this the same as writing?
```js
const generateTotalTimings = (words) =>{
     return   generatePreviousTimings(words[words.length - 1].start);
}
```

Reply:
ok, never-mind found it on [tackoverflow](https://stackoverflow.com/questions/39629962/arrow-function-without-curly-braces)